mw-webapp/src/router/pages.tsx

```diff
@@ -36,6 +37,10 @@ export const pages = {
     path: "users",
     element: <AllUsersPage />,
   },
+  redirect: {
+    path: "redirectPage",
+    element: <RedirectPage />,
+  },
```

Comment:
Pls add comment 
TODO: make one page Error for Error404, Redirect and etc

Reply:
done